packages/vscode-extension/src/project/metro.ts

```diff
@@ -440,10 +429,39 @@ export class Metro implements Disposable {
     return undefined;
   }
 
-  private async fetchDebuggerURL() {
-    // query list from http://localhost:${metroPort}/json/list
-    const list = await fetch(`http://localhost:${this._port}/json/list`);
-    const listJson = await list.json();
+  public async fetchRuntimeList(): Promise<CDPTargetDescription[] | undefined> {
```

Comment:
It doesn't list runtimes here but websocket targets. This is a more abstract concept as a single target actually may cover many runtimes. Specifically this happens when you do "reload JS" in react native – new runtime is created but it is listed under the same WS target.

I'd rename to `fetchCDPWebsocketTargets` or something simpler by removing "CDP" or "Websocket" fragments